services/brig/src/Brig/User/Auth/Cookie.hs

```diff
@@ -110,6 +110,30 @@ renewCookie old = do
     DB.insertCookie u old' (Just (DB.TTL (fromIntegral ttl)))
     return new
 
+-- | Whether a user has not renewed any of her cookies for longer than
+-- 'suspendCookiesOlderThanSecs'.  Call this always before 'newCookie', 'nextCookie',
+-- 'newCookieLimited' if there is a chance that the user should be suspended (we don't do it
+-- implicitly because of cyclical dependencies).
+mustSuspendInactiveUser :: UserId -> AppIO Bool
+mustSuspendInactiveUser uid = view (settings . to setSuspendInactiveUsers) >>= \case
+    Nothing -> pure True
```

Comment:
Without anything set on the config the users _always_ get suspended?

Reply:
yes.